test/config/ca.json

```diff
@@ -35,6 +35,7 @@
       "NumSessions": 2
     }],
     "expiry": "2160h",
+    "backdate": "1h",
```

Comment:
This should be left out. We would want the `ca.json` config to remain untouched to validate that updating the Boulder components with their in-place configurations won't cause problems.